templates/js-template-link/frameworks/runtime-src/proj.openharmony/CMakeLists.txt

```diff
@@ -7,14 +7,19 @@ set(CC_COCOS_SOURCES)
 set(CC_COMMON_SOURCES)
 set(CC_ALL_SOURCES)
 
-set(CC_USE_SE_V8 ON)
+set(CC_USE_SE_V8 OFF)
 set(CC_USE_SE_NAPI OFF)
+set(CC_USE_SE_JSVM ON)
+
+SET(CMAKE_BUILD_TYPE "Debug")
```

Comment:
Why do we need this setup?

Reply:
SET(CMAKE_BUILD_TYPE "Debug")

Reply:
This is redundant and needs to be deleted